updated top.css #footer height

arjunvijayvargiya opened the pull request:
changed footer attribute height:60px to min-height:60px; so that footer content overflow issue can be resolved

smcgregor commented:
@spectralsun Does this look good to you? You have done work on ensuring the apps fit into smaller widths, like the referenced mobile issue.

arjunvijayvargiya commented:
Just to be more clearer .I have added two more commits

1.changing modify equality operator to strict:https://github.com/arjunvijayvargiya/privly-applications/commit/6c4613bbf891a18cdd52023696c76e662d9f0914
2.updated Network.js to resolve privly-android issue:https://github.com/arjunvijayvargiya/privly-applications/commit/9a79304686395aabc48fc570df9a2660dc6e76dc

vshivam commented on shared/javascripts/network_service.js:
```diff
@@ -446,6 +446,7 @@ var privlyNetworkService = {
     $(".logged_out_nav").show();
     $(".injected_hide").show();
     privlyNetworkService.mobileHide();
+    androidJsBridge.showLoginActivity();
```
You should check if the application is running on the android platform. It does not make sense to do this on another platform.

arjunvijayvargiya replied:
This will take care of the platform in which the application is running.
var ua = navigator.userAgent.toLowerCase();
    var isAndroid = ua.indexOf("android") > -1; //&& ua.indexOf("mobile");
    if(isAndroid) {
     androidJsBridge.showLoginActivity();
    }

vshivam commented on shared/javascripts/tooltip.js:
```diff
@@ -204,7 +204,7 @@ var privlyTooltip = {
           // The rest of two columns are simetrical to the first two
           if(j <= 2) {
-            if(glyphArray[i * 3 + j] == "true") {
+            if(glyphArray[i * 3 + j] === "true") {
```
I am not too sure about this. @smcgregor will be able to comment better. It depends on the type of values the glyphArray stores.

smcgregor commented:
This has grown into three separate pull requests:

1. The footer style change.
2. The android login activity change.
3. A small stylistic change.

Can you please remove the commits for parts 2 and 3, and open separate pull requests for them?

smcgregor commented:
I'll merge this pull request as soon as it only have the footer change.

Also, I recommend doing all your work on separate feature branches. Your version of master should never be ahead of the main repo's version of master.
